ledger/eval.go

```diff
@@ -339,6 +340,11 @@ func (eval *BlockEvaluator) workaroundOverspentRewards(rewardPoolBalance basics.
 	return
 }
 
+// GetRoundTxIds returns a map of the transactions ids that we have for the given round
+func (eval *BlockEvaluator) GetRoundTxIds(round basics.Round) (txMap map[transactions.Txid]bool) {
```

Comment:
Can you expose this directly through `Ledger`?  It seems to be unrelated to the block evaluator, and you have access to the ledger at the call site.

Reply:
make sense. will do!